_README.md

```diff
@@ -0,0 +1,145 @@
+-->
+
+This template must be used with the AsyncAPI Generator. All available options on how to generate output are available [here](https://github.com/asyncapi/generator/).
+
+### CLI
+
+```bash
+npm install -g @asyncapi/generator
+ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.15/test/docs/dummy.yml https://github.com/asyncapi/template-for-generator-templates -o output
+open output/index.html
+```
+
+### Docker
+
+```bash
+docker run --rm -it \
+-v ${PWD}/output:/app/output \
+asyncapi/generator -o /app/output https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.15/test/docs/dummy.yml @asyncapi/markdown-template --force-write
+```
+
+## Template configuration
+
+<!--  
+This information should match the information provided in the template configuration under the `parameters` property.
+-->
+
+You can configure this template using different parameters. You pass them like this in the Generator CLI: `-p PARAM1_NAME=PARAM1_VALUE -p PARAM2_NAME=PARAM2_VALUE`
+
+
+Name | Description | Required | Default | Allowed Values | Example
+---|---|---|---|---|---
+asyncapiFileDir | Custom location of the AsyncAPI file that you provided as an input in a generation. | No | The root of the output directory. | Path to the custom directory relative to the root of output directory provided as a string | `/custom/dir`
+pdf | Generate PDF from initially generated HTML file | No | `false` | `true`/`false` | `true`
+png | Generate PNG with a schemas relations diagram. | No | `false` | `true`/`false` | `true`
+svg | Generate SVG with a schemas relations diagram. | No | `false` | `true`/`false` | `true`
+maxTextSize | It is possible that in case of an extremely large AsyncAPI document default mermaid recommended text size will not be enough. Then you need to make it larger explicitly. | No | `50000` |  Number higher than 50000 | `70000`
+
+## Custom hooks that you can disable
+
+<!--  
+Document hooks that users can disable and template will still work as expected. Remember that a specific hook can be disabled only if it has a name. In other words, make sure your hook functions are not anonymous.
+-->
+
+Functionality of this template is extended with different hooks that you can disable. You disable them like this in the Generator CLI: `-d HOOK_TYPE1=HOOK_NAME1 -d HOOK_TYPE1=HOOK_NAME2 -d HOOK_TYPE2`
+
+
+Type | Name | Description
+---|---|---|---|---|---
+generate:after | createAsyncapiFile | It creates AsyncAPI file with content of the spec file passed to the generator
+
+## Development
+
+<!--  
+This section will look the same everywhere, just make sure it references your template.
+-->
+
+The most straightforward command to use this Template is:
```

Comment:
Is it `Template` or `Template`?  (capitalized vs not capitalized) -> Is `Template` here something different, like an Object? I can see it appears later in the text.

Reply:
yeah, should be `template` as it is something I would compare to words like `repository` or `project`, something specific but not unique, like Generator. Changing it now